OpenRA.Game/Game.cs

```diff
@@ -100,6 +100,11 @@ static void JoinLocal()
 		public static long RunTime => stopwatch.ElapsedMilliseconds;
 
 		public static int RenderFrame = 0;
+
+		// Needed for interpolation
+		public static long LastWorldTickRunTime = 0;
+		public static int MsSinceLastWorldTick = 0;
```

Comment:
IMO it would be better if we could call this something like `RenderInterpolatedTimestepProgress` to make its purpose and use in `WPos.Lerp(pos, nextPos, Game.RenderInterpolatedTimestepProgress, world.Timestep);` clearer.